client/my-sites/customer-home/main.jsx

```diff
@@ -272,14 +274,25 @@ class Home extends Component {
 								label={ translate( 'Design a logo' ) }
 								iconSrc="/calypso/images/customer-home/logo.svg"
 							/>
-							<ActionBox
-								onClick={ () => {
-									trackAction( 'my_site', 'add_gsuite' );
-									page( `/email/${ siteSlug }` );
-								} }
-								label={ translate( 'Add G Suite' ) }
-								iconSrc="/calypso/images/customer-home/gsuite.svg"
-							/>
+							{ hasCustomDomain ? (
+								<ActionBox
+									onClick={ () => {
+										trackAction( 'my_site', 'add_gsuite' );
+										page( `/email/${ siteSlug }` );
+									} }
+									label={ translate( 'Add email' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Add Email' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).